DataCollection/src/Infrastructure/Persistence/DataCollectionContext.cs

```diff
@@ -60,18 +61,9 @@ public class DataCollectionContext : DbContext, ILocationPersistence, IWorkLogPe
 
         public IEnumerable<BirdSpecies> GetBirdSpeciesBySurveyTypeId(int surveyTypeId)
         {
-            var returnVal = new List<BirdSpecies>();
-            var surveyType = SurveyTypes.Include(survey => survey.BirdSpecies).FirstOrDefault(surveyItem => surveyItem.Id == surveyTypeId);
-
-            if (surveyType != null)
-            {
-                if (surveyType.BirdSpecies != null)
-                {
-                    returnVal = surveyType.BirdSpecies.ToList();
-                }
-            }
-            return returnVal;
+            throw new NotImplementedException();
```

Comment:
I believe that the DataCollectionContext class will still inherit from IBirdSpeciesPersistence interface?